### DISCUSSION TITLE
Permission to see general and shout messages in chat, without taking the ability to write general and shout messages.

### DISCUSSION THREAD

**Discussion by VitSun2001** [Zrips/CMI#5019] [Enhancement]
There is a 'cmi.chat.rangebypass' permission that allows people with it to see and send messages from any distance on the server. 
However, this can become inconvenient to use for supervision of general(local) chat to prevent abuse and server rule violation. This is because it removes the ability to write in general(local) chat (if it has range restriction in config).
Chat differentiation is very useful if server moderators/administrators need to convey some information for nearby players but not to all the server.

Summary: Add a permission or command that allows spying on general(local) chat across the server without messing with ability to sent general(local) and shout(global) messages separately.

**Comment by div2005** [comment]
up
